app/assets/stylesheets/vars/_utils.scss

```diff
@@ -0,0 +1,37 @@
+// Border radius
+$br-default: 3px;
+
+// Break points
+$breakpoint-s: 640px;
+$breakpoint-m: 768px;
+$breakpoint-l: 1024px;
+$breakpoint-xl: 1280px;
```

Comment:
I'm not yet sold on these breakpoints.. I've put them here for now more as a placeholder.